### PR TITLE
Implemented User Interface tab to Modeler

### PR DESCRIPTION
**Pull request by zebilha**
To explain Lists and Grids column limit changes

### CODE REVIEW

File: docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md [hunk 1]
```diff
@@ -103,46 +111,49 @@ In the following sample, to access the value representing the attribute _TotalAm
...
+In this sample, custom element _supplier_ value is changed:
 
 ```JavaScript
     this.supplier = "S001";
 ```
 
 ### How to add or remove a message?
-In this sample, is removed and added (based on a condition) a message to the base element *_code*:
+
+In this sample, is removed and added (based on a condition) a message to the base element _\_code_:
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md:156]: In this sample a message is removed, and added, to the base element _\_code_:

File: docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md [hunk 2]
```diff
@@ -151,47 +162,47 @@ In this sample, is removed and added (based on a condition) a message to the bas
     else
         this._metadata.elements._code.addMessage('My success message', 'success', 'validation');
 ```
-*Note*: There are available only two types of message, as shown in the previous sample: **error** and **success**.
+
+_Note_: There are available only two types of message, as shown in the previous sample: **error** and **success**.
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md:166]: _Note_: There are only two types of messages available, as shown in the previous sample: **error** and **success**.

File: docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md [hunk 3]
```diff
@@ -242,48 +253,53 @@ In this sample, the container *myContainer* is set as opened:
 ### How to inform the user that an element is loading data?
 
 Inputs give you an attribute to control an internal loader. This is particularly useful when a triggered C# behaviour will change the value of other elements and we want to inform the user that something is happening relative to that element.
-You will be responsible to enable and disable the loader. For example, you can enable when a given element changes the value and disable it [in the UI After Change behaviour](omnia3_modeler_uibehaviours.html#2-types-of-behaviours). 
+You will be responsible to enable and disable the loader. For example, you can enable when a given element changes the value and disable it [in the UI After Change behaviour](omnia3_modeler_uibehaviours.html#2-types-of-behaviours).
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md:256]: Example: enable, or disable, it when a given element changes its value

File: docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md [hunk 4]
```diff
@@ -242,48 +253,53 @@ In this sample, the container *myContainer* is set as opened:
...
 ```JavaScript
     this._metadata.elements._description.attributes.isLoading = true;
 ```
 
 ### **Decimal Attributes**
+
 ### How to change the number of decimal places of the element?
-In this sample, the number of decimal places of the element *decimalField* is setted to a custom value *(3)*. 
+
+In this sample, the number of decimal places of the element _decimalField_ is setted to a custom value _(3)_.
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md:266]: In this sample, the number of decimal places of the element _decimalField_ is set to a custom value _(3)_.

File: docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md [hunk 5]
```diff
@@ -242,48 +253,53 @@ In this sample, the container *myContainer* is set as opened:
...
 
 ```JavaScript
     this._metadata.elements.decimalField.attributes.precision = 3;
 ```
 
-*Note 1: This is only possible in elements that represent decimal attributes. In the other data types, even this attribute is setted, the value will be ignored.*
+_Note 1: This is only possible in elements that represent decimal attributes. In the other data types, even this attribute is setted, the value will be ignored._
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md:272]: Não percebi a segunda frase @zebilha , o que querias dizer?

Comment (zebilha) [reply docs/pages/omnia3/Modeler/omnia3_modeler_entities_UI.md:272]: This change was a syntax only change that automatically occurred when saving the file.

**Pull request merged**